src/cloud/project/project-enable-mfa-enforcement.md

```diff
@@ -4,14 +4,21 @@ title: Enable MFA enforcement for SSH access
 functional_areas:
   - Cloud
   - Configuration
+  - Security
+  - Compliance
+  
+redirect_from:
+  - /cloud/project/project-enable-mfa-project.html
 ---
 
-For added security, {{site.data.var.ece}} provides a mutlitfactor authentication (MFA) enforcement feature to manage multifactor authentication requirements for SSH access to Cloud infrastructure.
+{%include cloud/beta-release-content-warning.md%}
 
-When MFA enforcement is enabled on a project, any user or machine account establishing an SSH connection to an environment must follow an authentication workflow that requires a two-factor authentication (2FA) code and an SSH certificate, or an API token to access the environment.
+For added security, {{site.data.var.ece}} provides a mutlitfactor authentication (MFA) enforcement feature to manage authentication requirements for SSH access to Cloud environments.
```

Comment:
There’s a typo  “mutlitfactor authentication” in the first sentence. Also the word "feature" isn't necessary. It's just a filler word.
```suggestion
For added security, {{site.data.var.ece}} provides mutltifactor authentication (MFA) enforcement to manage authentication requirements for SSH access to Cloud environments.
```